FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -593,6 +595,10 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
  */
 - (void)verifyClientWithUIDelegate:(nullable id<FIRAuthUIDelegate>)UIDelegate
                         completion:(FIRVerifyClientCallback)completion {
+  if ([GULAppEnvironmentUtil isSimulator]) {
```

Comment:
Should there be an iOS version check for at least 16 as well?

Reply:
Yeah, good point.

Reply:
Added a check to not include this change in unit tests (as they're run in the simulators). Please take a look. Thanks!